examples/usecase-todo/package.json

```diff
@@ -7,16 +7,15 @@
     "dev": "keystone dev",
     "start": "keystone start",
     "build": "keystone build",
-    "postinstall": "keystone postinstall",
     "seed-data": "tsx seed-data.ts"
   },
   "dependencies": {
     "@keystone-6/core": "^6.2.0",
-    "@prisma/client": "5.17.0"
+    "@prisma/client": "*"
   },
   "devDependencies": {
-    "prisma": "5.17.0",
-    "tsx": "^4.0.0",
+    "prisma": "*",
```

Comment:
Won't this result in potentially incompatible versions running on CodeSandbox?

Reply:
This seems to solve a couple of issues, especially when combining with removing and untracking the schema files.

It means that it's always respecting the version from `keystone/core` - this is one of the reasons code sandbox isn't working at the moment.

After fixing the prisma URL issue, the next issue was schemas out of sync, because the current version of keystone/core that gets installed (6.1), uses prisma 5.13 - since 5.13 the generator has changed a little.